src/main/kotlin/lotto/AutoLottoMaker.kt

```diff
@@ -2,26 +2,16 @@ package lotto
 
 class AutoLottoMaker : LottoMaker {
 
-    private val lotto: MutableList<LottoNumber> = mutableListOf()
+    override fun make(lottos: List<LottoNumber>): Lotto {
 
-    init {
-        for (i in LOTTO_NUMBER_START..LOTTO_NUMBER_END) lotto.add(CachedLottoNumbers.getLottoNumber(i))
-    }
-
-    override fun make(): Lotto {
-        return Lotto(get().toSet())
-    }
-
-    private fun get(): List<LottoNumber> {
-        return lotto.toList()
+        val lottoNumbers = lottos
             .shuffled()
             .take(LOTTO_SIZE)
             .sortedBy { it.number }
+        return Lotto(lottoNumbers.toSet())
```

Comment:
let과 메서드참조를 활용하면 변수 없이 Lotto객체를 리턴할 수 있어요
```suggestion
        lottos
            .shuffled()
            .take(LOTTO_SIZE)
            .sortedBy { it.number }
            .toSet()
            .let(::Lotto)
```

Reply:
오~ 감사합니다 let 을 활용 잘 해 보아야겠어요!